sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -2267,23 +2110,16 @@ Mono<Response<SubscriptionProperties>> updateSubscriptionWithResponse(Subscripti
     Mono<Response<TopicProperties>> updateTopicWithResponse(TopicProperties topic, Context context) {
         if (topic == null) {
             return monoError(LOGGER, new NullPointerException("'topic' cannot be null"));
-        } else if (context == null) {
-            return monoError(LOGGER, new NullPointerException("'context' cannot be null."));
         }
```

Comment:
Just to make sure, in all the calling methods, the context is never null?

Reply:
yes, correct. Since `context` being not null isn't a user requirement but an implementation detail.